start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloReloadingVFSClassLoader.java

```diff
@@ -183,12 +184,23 @@ public AccumuloReloadingVFSClassLoader(String uris, FileSystemManager vfs,
     monitor.setDelay(monitorDelay);
     monitor.setRecursive(false);
     for (FileObject file : pathsToMonitor) {
-      monitor.addFile(file);
+      addFileToMonitor(file);
       log.debug("monitoring {}", file);
     }
     monitor.start();
   }
 
+  private void addFileToMonitor(FileObject file) {
+    try {
+      monitor.addFile(file);
+    } catch (RuntimeException re) {
+      if (re.getMessage().contains("files-cache"))
+        log.error("There is no implementation for files-cache in VFS2");
```

Comment:
Nice. Turn the exception into a log message and move on. I like this strategy. It will keep things working, but not prevent us from being able to debug.

One suggestion: please include the stack trace in the log message:

```suggestion
        log.error("files-cache error adding {} to VFS monitor. There is no implementation for files-cache in VFS2", file, re);
```